docs/pages/guides/using-hermes.mdx

```diff
@@ -126,21 +56,53 @@ You may want to use Hermes on one platform and JSC on another. One way to do thi
-
-## Limitations
+### Troubleshooting
+
+> **warning** Get `No compatible apps connected. JavaScript Debugging can only be used with the Hermes engine.` when opening the debugger.
+
+- Make sure you [setup Hermes in the `jsEngine` field](#setup).
+- If your app is built by `eas build`, `npx expo run:android` or `npx expo run:ios`, make sure it is debug build.
+- Internally, the app will establish a WebSocket connection, make sure your app is connected to the development server.
+
+  - You could try to reload the app by pressing <kbd>r</kbd> in the Expo CLI Terminal UI.
+  - You could test the debugging availability by the command: `curl http://127.0.0.1:19000/json/list` (adjust the `127.0.0.1:19000` to match your dev server URL). The HTTP response should not be an empty array but something like this:
+
+  ```json
+  [
+    {
+      "id": "0-2",
+      "description": "host.exp.Exponent",
+      "title": "Hermes ABI47_0_0React Native",
+      "faviconUrl": "https://reactjs.org/favicon.ico",
+      "devtoolsFrontendUrl": "devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws=%5B%3A%3A1%5D%3A19000%2Finspector%2Fdebug%3Fdevice%3D0%26page%3D2",
+      "type": "node",
+      "webSocketDebuggerUrl": "ws://[::1]:19000/inspector/debug?device=0&page=2",
+      "vm": "Hermes"
+    },
+    {
+      "id": "0--1",
+      "description": "host.exp.Exponent",
+      "title": "React Native Experimental (Improved Chrome Reloads)",
+      "faviconUrl": "https://reactjs.org/favicon.ico",
+      "devtoolsFrontendUrl": "devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws=%5B%3A%3A1%5D%3A19000%2Finspector%2Fdebug%3Fdevice%3D0%26page%3D-1",
+      "type": "node",
+      "webSocketDebuggerUrl": "ws://[::1]:19000/inspector/debug?device=0&page=-1",
+      "vm": "don't use"
+    }
+  ]
+  ```
 
-### Standalone apps created with `expo build` are not supported
+### Can I use Remote Debugging on Hermes?
 
-The classic build system [isn't flexible enough](https://blog.expo.dev/expo-managed-workflow-in-2021-5b887bbf7dbb) to support using Hermes for some apps and not for others. You will need to use the new build system, [EAS Build](/build/introduction), to use Hermes in your standalone apps.
+A limitation with [Remote Debugging](/workflow/glossary-of-terms/#remote-debugging) is that it does not work with modules built on top of [JSI](https://github.com/react-native-community/discussions-and-proposals/issues/91). For example, if your app uses [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) version 2, [Remote Debugging will not work](https://docs.swmansion.com/react-native-reanimated/docs/#known-problems-and-limitations). Hermes supports [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/v8/) to debug JavaScript in place and makes it possible to debug your app even when using JSI modules, so Remote Debugging is not available when using Hermes.
```

Comment:
```suggestion
One of the many limitations of [remote debugging](/workflow/glossary-of-terms/#remote-debugging) is that it does not work with modules built on top of [JSI](https://github.com/react-native-community/discussions-and-proposals/issues/91). For example, if your app uses [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) version 2 or higher, [remote debugging will not work](https://docs.swmansion.com/react-native-reanimated/docs/#known-problems-and-limitations). 

Hermes supports [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/v8/) to debug JavaScript in place by connecting to the running engine on the device. This enables you to debug your app even when using JSI modules. This debugging technique is superior to remote debugging and replaces it for apps using Hermes.
```

the intention of the wording changes here is to clarify that you're not missing out on anything by moving towards the chrome devtools protocol instead of remote debugging